cumulusci/tasks/bulkdata.py

```diff
@@ -579,7 +683,7 @@ def _init_db(self):
         self.metadata.bind = self.engine
 
         # Create the tables
-        self._create_tables()
+        self._create_tables(create=True)
```

Comment:
A function named `_create_tables` that takes a boolean `create` param reads really strangely. What's the actual distinction between its two modes of operation and can we give it a better name based on that?